Java/core/src/test/java/com/amazon/randomcutforest/state/store/NodeStoreMapperTest.java

```diff
@@ -22,35 +22,33 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-import com.amazon.randomcutforest.store.SmallNodeStore;
+import com.amazon.randomcutforest.store.NodeStore;
 
 public class NodeStoreMapperTest {
-    private SmallNodeStoreMapper mapper;
+    private NodeStoreMapper mapper;
 
     @BeforeEach
     public void setUp() {
-        mapper = new SmallNodeStoreMapper();
+        mapper = new NodeStoreMapper();
     }
 
     @Test
     public void testRoundTrip() {
-        SmallNodeStore store = new SmallNodeStore((short) 10);
+        NodeStore store = new NodeStore((short) 10);
```

Comment:
I think we don't have to cast to `short` any more.

Reply:
Done.